src/ResourceManager/Profile/Commands.Profile/Context/SaveAzureRMContext.cs

```diff
@@ -26,8 +26,10 @@ namespace Microsoft.Azure.Commands.Profile
     /// <summary>
     /// Saves Microsoft Azure profile.
     /// </summary>
-    [Cmdlet(VerbsData.Save, "AzureRmProfile", SupportsShouldProcess = true), OutputType(typeof(PSAzureProfile))]
-    public class SaveAzureRMProfileCommand : AzureRMCmdlet
+    [Cmdlet(VerbsData.Save, "AzureRmContext", SupportsShouldProcess = true), OutputType(typeof(PSAzureProfile))]
+    [Alias("Save-AzureRmProfile")]
+    [Obsolete("Save-AzureRmProfile will be renamed to Save-AzureRmContext in the next release.", false)]
+    public class SaveAzureRMContextCommand : AzureRMCmdlet
```

Comment:
No, the analyzer is complaining about the ValidateNotNullOrEmpty attribute added to the path parameter.  Since I added an alias to the old cmdlet name, the cmdlet name change is not a breaking change (and the tool doesn't consider it to be one).